.github/workflows/blade-pr-title-check.yml

```diff
@@ -0,0 +1,40 @@
+name: Blade PR Title Check
+
+on: 
+  pull_request:
+     types: [open, edited, synchronize]
+
+concurrency: # cancel previously running workflows when a new workflow is re-run
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pr-title-check:
+    name: Check
+    runs-on: ubuntu-latest # nosemgrep: non-self-hosted-runner
+    env:
+      GITHUB_TOKEN: ${{ secrets.CI_BOT_TOKEN }}
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        id: pr_title_check
+      - uses: marocchino/sticky-pull-request-comment@v2
+        # When the previous steps fails, the workflow would stop. By adding this
+        # condition you can continue the execution with the populated error message.
+        if: always() && (steps.pr_title_check.outputs.error_message != null)
+        with:
+          header: pr-title-comment
+          message: |            
+            We follow the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/) and it looks like your PR Title doesn't fit the specification.
+
+            Details:
+            
+            ```
+            ${{ steps.pr_title_check.outputs.error_message }}
+            ```
+      # Delete a previous comment when the issue has been resolved
+      - if: ${{ steps.pr_title_check.outputs.error_message == null }}
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:   
+          header: pr-title-comment
+          message: '✅ PR Title follows [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/).'
```

Comment:
I'd prefer just doing `delete: true` but for some reason that's not working. Tried debugging for a while but using this as a workaround for now